src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "cinema_hall")
```

Comment:
```suggestion
@Table(name = "cinema_halls")
```
Table name should be plural